addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -425,19 +426,19 @@ def l10n_in_grouping_key_generator(tax_values):
             base_line = tax_values["base_line_id"]
             tax_line = tax_values["tax_line_id"]
             line_code = "other"
-            if any(tag in tax_line.tax_tag_ids for tag in self.env.ref("l10n_in.tax_report_line_cess").sudo().tag_ids):
+            if tax_line.tax_tag_ids in self.env.ref("l10n_in.tax_tag_cess"):
```

Comment:
```suggestion
            if self.env.ref("l10n_in.tax_tag_cess") in tax_line.tax_tag_ids:
```

I guess this makes more sense, right ?